parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -676,35 +676,37 @@ private static final int compareByteArrayToByteBuffer(byte[] array1, int offset1
     if (array1 == null && buf == null) return 0;
```

Comment:
lets rename all the binary compare methods to have either signed or unsigned in their names to avoid further confusion.